x/cork/client/cli/query.go

```diff
@@ -18,7 +19,11 @@ func GetQueryCmd() *cobra.Command {
 
 	allocationQueryCmd.AddCommand([]*cobra.Command{
```

Comment:
Realizing this still says `allocationQueryCmd`

Reply:
fixed